tests/std/tests/P0355R7_calendars_and_time_zones_formatting/test.cpp

```diff
@@ -1061,7 +1063,39 @@ void test_locale() {
     assert(stream(year_month_weekday_last{2021y / May / Tuesday[last]}) == STR("2021/Mai/Di[last]"));
 }
 
+template <typename T>
+concept ostream_insertable = requires(std::ostream& o, const T& t) { o << t; };
+
+template <typename Dur>
+void check_stream_insertion_operator_for_duration() {
+    if constexpr (ostream_insertable<sys_time<Dur>>) {
+        std::cout << sys_time<Dur>{};

```

Comment:
This test is already `using namespace std;`. Additionally, we generally print to an ostringstream where we can programmatically inspect the output, instead of `cout`.